File: app/src/main/java/com/overetch/kample/ActivityUtils.java
package com.overetch.kample;

import android.support.annotation.NonNull;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;

/**
 * Created by Igor.Sakovich on 24.01.2017.
 */

public class ActivityUtils {
    public static void addFragmentToActivity (@NonNull FragmentManager fragmentManager,
                                              @NonNull Fragment fragment) {
        FragmentTransaction transaction = fragmentManager.beginTransaction();
        transaction.add(R.id.contentFrame, fragment);
        transaction.commit();
    }
}

File: app/src/main/java/com/overetch/kample/utils/ActivityUtils.kt
package com.overetch.kample.utils

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import com.overetch.kample.R

/**
 * Created by Igor.Sakovich on 24.01.2017.
 */

object ActivityUtils {
    fun addFragmentToActivity(fragmentManager: FragmentManager,
                              fragment: Fragment,
                              addToBackStack: Boolean = false) {
        val transaction = fragmentManager.beginTransaction()
        transaction.add(R.id.contentFrame, fragment)
        if (addToBackStack)
            transaction.addToBackStack(null)
        transaction.commit()
    }
}
